### DISCUSSION TITLE
IP restrictions, multiple IPs?

### DISCUSSION THREAD

**Discussion by mattwalshdev** [planetscale/discussion#614] [Q&A] [answered]
IP restrictions documentation is a little light on https://planetscale.com/docs/concepts/connection-strings#ip-restrictions
I see you can include single or multiple in the same subnet using the slash notation, but can you include multiple in different areas? If so what's the format? Semicolon break?
eg. 200.200.200.00/24;100.100.100.00/32

**Comment by mscoutermarsh** [comment, marked as answer]
Hi @mattwalshdev. Keep clicking `add`, it will let you add multiple.
